e2e_tests/tests/fixtures/configuration/run_fork.py

```diff
@@ -0,0 +1 @@
+print("forked")
```

Comment:
similar to a comment in another PR, can we please rename it `configuration` -> `generic_task` for clarity.